assets/src/console/pages/products/ListPage.tsx

```diff
@@ -24,6 +25,35 @@ export interface props {
   gatewayAccount: GatewayAccountFragment;
 }
 
+const Ul = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+`;
+
+const Li = styled.li`
+  margin: 1rem 0 3rem;
+  ${P} {
+    font-weight: 700;
+    margin-bottom: 0;
+  }
+`;
+
+const CustomLink = styled(Link)`
```

Comment:
@acodegirl why did you decide to go with `CustomLink` rather than `DangerousLink`?